db/models/cart.js

```diff
@@ -0,0 +1,82 @@
+const client = require("./client");
+
+async function addItemToCart({ orderId, quantity }) {
+    try {
+        const { rows: [newOrder] } = await client.query(`
+        INSERT INTO cart("orderId", quantity)
+        VALUES ($1, $2)
+        RETURNING *;
+        `, [orderId, quantity]);
+
+        return newOrder;
+    } catch (error) {
+        console.error("Could not add item to order");
+        throw error;
+    }
+}
+
+async function getOrderById(id) {
+    try {
+        const { rows: [cart] } = await client.query(`
+        SELECT *
+        FROM cart
+        WHERE id=$1;
+        `, [id]);
+
+        return cart;
+    } catch (error) {
+        console.error("Could not get order by id");
+        throw error;
+    }
+}
+
+async function updateCart({ id }) {
+    try {
+        const order = await getOrderById(id);
+        const fields = {};
```

Comment:
fields here is undefined. Then at line 40, fields is called to create the setString. In this scenario, setString will always be undefined and hit your return at 44.

Reply:
line 33 function arguments:
```suggestion
        {id, ...fields}
```